src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.cs

```diff
@@ -39,17 +40,20 @@ internal partial class FormatCommandHandler :
     {
         private readonly ITextUndoHistoryRegistry _undoHistoryRegistry;
         private readonly IEditorOperationsFactoryService _editorOperationsFactoryService;
+        private readonly IWaitIndicator _waitIndicator;
```

Comment:
I dont think we need this.

Reply:
@heejaechang This is necessary.  Otherwise we cannot display updated messages in teh UI, or progress to the user.